lib/dom-helpers.mjs

```diff
@@ -0,0 +1,19 @@
+/**
+ * Creates an element with the given attributes
+ *
+ * TODO: Extend to allow passing props and children if necessary
+ * TODO: Make available to components once support for older browsers is dropped (add tests)
+ *
+ * @param {String} tagName
+ * @param {Object} attributes
+ * @returns HTMLElement
+ */
+export function createElement (tagName, attributes = {}) {
+  const el = document.createElement(tagName)
```

Comment:
As flagged in Slack, although `document` seems to be part of the default globals for Standard, we might want to explicitly configure eslint so that it knows this is file is in the context of a browser / jsdom (for example by adding another override for this file in `.eslintrc.js`)